how-to/how-to-template-guide.md

```diff
@@ -0,0 +1,149 @@
+# How-to guide
+
+information_source: Read this document before you start working on the [how-to template](how-to-template.md).
+
+## Introduction
+
+How-to guide takes your users through a series of steps required to solve a specific problem. It shows your users how to solve a real-world problem or complete a task in your application, like how to create an issue in GitHub.
+
+The how-to guide clearly describes a set of ordered sequential steps your users must complete to accomplish a task. The how-to guide assumes that a user has basic knowledge of the application and has already read the quickstart and the tutorial.
+
+Do not use how-to guides to teach concepts.
+
+How-to guides are often confused with tutorials. How-to guides are task-oriented, while tutorials are learning-oriented. The table below identifies the differences between the two.
+
+| Tutorials                                                                                                                                       | How-to                                                                                                   |
+| ----------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| Learning oriented                                                                                                                               | Task oriented                                                                                            |
```

Comment:
Learning oriented / task oriented
In my opinion, this items are not self explanatory enough. Can be definitely combined with the next row in a manner like Learning oriented: helps beginners or expert users learn a new feature and provide a successful learning experience.